tests/installation/bootloader_svirt.pm

```diff
@@ -236,11 +237,17 @@ sub run {
     if ($vmm_family eq 'kvm') {
         $iface_model = 'virtio';
     }
-    elsif ($vmm_family eq 'xen' && $vmm_type eq 'hvm') {
-        $iface_model = 'netfront';
+    elsif ($vmm_family eq 'xen') {
+        $ifacecfg{type}        = 'bridge';
+        $ifacecfg{source}      = {bridge  => 'br0'};
+        $ifacecfg{virtualport} = {type    => 'openvswitch'};
+        $ifacecfg{mac}         = {address => genmac('00:16:3e')};
+        $iface_model           = 'netfront';
```

Comment:
Just an idea but you might like to use test_data here

Reply:
Doesn't that mean all the VM environment details would need to be shifted into `yaml` schedule files?

Reply:
let me check but i believe no.  you should need only a test_data yaml and load it in the module

Reply:
That sounds as we would refactor this module in a certain way. OTOH [test_data](https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/test_data) (not sure if you mean it?) seems to be meant to be used as validation data for the SUT. In this case we are specifying VM details for XML files that are used by `libvirt`.

Reply:
ignore. it doesnt work.